joplin/base/models.py

```diff
@@ -70,10 +103,12 @@ def janis_preview_url(self):
 
         return os.environ["JANIS_URL"] + "/en/preview/" + url_page_type + "/" + global_id
 
+    class Meta:
+        abstract = True
 
-class ServicePage(Page, JanisPage):
-    created_at = models.DateTimeField(auto_now_add=True)
-    updated_at = models.DateTimeField(auto_now=True)
+JanisPage._meta.get_field('title').verbose_name='Write an actionable title'
```

Comment:
So this line seems like it is composed outside the scope of the class definition. I'm no 🐍 expert but could we do something similar inside the JanisPage class using something like the `self` keyword? 

Also, it seems like an assumption (perhaps a good one) that all our content pages would have actionable titles. But I see that not holding up as we're already talking about Press Releases and Informational page. Might be worth a quick Slack to a content team member to confirm this assumption.

Reply:
yea, the problem here is that the title field is initialized in the parent Page object as a field of type CharField -- so I can't just set self.title with a simple string -- it's a specialized CharField class.  I thought about redefining title within the JanisPage model as
```
title = models.CharField(
        verbose_name=_('Write an actionable title'),
        max_length=255,
        help_text=_("The page title as you'd like it to be seen by the public")
)
```
but I thought if the parent definition changed in wagtail in some update, that redefining the title entirely wasn't as update friendly. But I can definitely try this.
But I can reset the verbose name as above, because this line is executed after class definition.

Reply:
Seems like django's not a fan of overriding fields in child classes when the parent isn't an abstract model (Page is not abstract) -- so I'm just going with the suggested approach [here](https://stackoverflow.com/questions/927729/how-to-override-the-verbose-name-of-a-superclass-model-field-in-django)